### DISCUSSION TITLE
how to use $topN ?

### DISCUSSION THREAD

**Discussion by vnugent** [Automattic/mongoose#12026] [Q&A] [answered]
Is `$topN` accumulator available for $group?

I'm getting   Type '{ $topN: {}; }' is not assignable to type 'AccumulatorOperator'.

https://jira.mongodb.org/browse/SERVER-9377

**Comment by vkarpov15** [comment, marked as answer]
Fix will be in v6.4.3, sorry for the inconvenience!

**Reply by vkarpov15** [reply]
See https://github.com/Automattic/mongoose/issues/12053
